fibonacci/julia/code.jl

```diff
@@ -1,15 +1,5 @@
-# base case 1
-fibonacci(::Val{0}) = 0
-# base case 2
-fibonacci(::Val{1}) = 1
-# general case
-fibonacci(::Val{n}) where n = fibonacci(Val(n-1)) + fibonacci(Val(n-2))
+fibonacci(n) = n < 2 ? n : fibonacci(n-1) + fibonacci(n-2)
```

Comment:
This needs to go back to having an if / case for 0 and 1.

Reply:
Is `(n == 0 || no == 1) ? n : fibonacci(n-1) + fibonacci(n-2)` okay, or does it need to be the full

```julia
n == 0 ? 0 :
    n == 1 ? 1 :
    fibonacci(n-1) + fibonacci(n-2)
```